plugin/build.gradle.kts

```diff
@@ -49,13 +49,12 @@ val tooling: Configuration by configurations.creating {
 }
 
 dependencies {
-    implementation(compose.desktop.linux_arm64)
-    implementation(compose.desktop.linux_x64)
-    implementation(compose.desktop.macos_arm64)
-    implementation(compose.desktop.macos_x64)
-    implementation(compose.desktop.windows_x64)
+    compileOnly(compose.desktop.linux_arm64)
```

Comment:
I believe that we can remove the runtimes altogether. Isn't the preview artifact available separately?

Reply:
found here https://mvnrepository.com/repos/google
[lib](https://mvnrepository.com/artifact/androidx.compose.ui/ui-tooling-preview) but not sure if need something else, i'm not able to import it correctly

solved by loading compose.desktop.currentOS